targets/TARGET_STM/qspi_api.c

```diff
@@ -192,7 +192,8 @@ qspi_status_t qspi_init(qspi_t *obj, PinName io0, PinName io1, PinName io2, PinN
 qspi_status_t qspi_free(qspi_t *obj)
 {
     // TODO
-    return QSPI_STATUS_ERROR;
+    //return QSPI_STATUS_ERROR;
+    return QSPI_STATUS_OK;
```

Comment:
What's the reason, this function can't be fully implemented but required a temperate fix?

Reply:
I think it's temporary fix. It should be implemented

@adustm Please review